docker-compose/run_integration_tests_local_keybase.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
```

Comment:
Might be good to add a safety command like `set -euo pipefail` here.

Reply:
> set -euo pipefail
Good idea!  That actual helped to identify a few issues in the script. Just updated this.